src/Service/Functions.php

```diff
@@ -102,4 +102,22 @@ public function getFunctionNamesByParameter(string $possiblyParameter): iterable
             }
         }
     }
+
+    public function getFunctionNamesByAllParameters(array $possiblyParameters): iterable
+    {
+        $sortedPossibleParameters = $this->sort($possiblyParameters);
+
+        foreach ($this->functionParameters as $functionName => $parameters) {
+            if ($this->sort($parameters) === $sortedPossibleParameters) {
+                yield $functionName;
+            }
+        }
+    }
+
+    private function sort(array $array): array
```

Comment:
Why own private function without any other meaning when you can call `sort` directly? 🤔

Reply:
because `sort` is sorting by reference and cannot be chained